operating/manage-user-access/configure-gsuite-identity.adoc

```diff
@@ -0,0 +1,15 @@
+[id="configure-gsuite-identity"]
+= Configuring G Suite as an OIDC identity provider
```

Comment:
Just checking that "G Suite" is the correct terminology, because when I click on the link in this assembly, it goes to a page titled "Google Workspace".